tests/lib/rules/no-unneeded-ternary.js

```diff
@@ -57,6 +57,16 @@ ruleTester.run("no-unneeded-ternary", rule, {
             }]
         },
         {
+            code: "var a = x ? true : false;",
+            output: "var a = !!(x);",
```

Comment:
why are the parens output when they're not necessary? or are you delegating to no-extra-parens here